src/components/PostList/PostList.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import './PostList.scss';
+import { Post } from '../Post/Post';
+import { typePost } from '../../types';
+
+export const PostList = ({ posts }) => (
+  <div className="postList">
+    {posts.map(post => (
+      <div key={post.id}>
+        <Post post={post} />
+      </div>
+    ))}
```

Comment:
if you need wrapper for each post, add it to the Post component

```suggestion
    {posts.map(post => (
        <Post post={post} key={post.id} />
    ))}
```